### PR TITLE
changed database and entities

### PR DESCRIPTION
**Pull request by zoegrippo**

### CODE REVIEW

File: apolloServer/src/main/java/com/apollo/com/apollo/entities/Strategy.java [hunk 1]
```diff
@@ -4,14 +4,53 @@
...
-    @Column(name="onoff") private boolean onoff;
+    @Column(name="strategyName") private String strategyName;
+    @Column(name="isRunning") private boolean isRunning;
+    @Column(name="startingVol") private Integer startingVol;
+    @Column(name="stock") private String stock; // stock ticker symbol
+    @Column(name="exitProfitPercent") private Double exitProfitPercent;
+    @Column(name="exitLossPercent") private Double exitLossPercent;
+    @Column(name="stdDevs") private Double stdDevs; // bollinger bands
+    @Column(name="shortTime") private Integer shortTime; // two moving averages. short time also used for other two strategies
+//    @Column(name="longTime") private Integer longTime;
```
Comment (robtrujillo) [inline apolloServer/src/main/java/com/apollo/com/apollo/entities/Strategy.java]: remove commented out code

Comment (zoegrippo) [reply apolloServer/src/main/java/com/apollo/com/apollo/entities/Strategy.java]: done

File: apolloServer/src/main/java/com/apollo/com/apollo/entities/Strategy.java [hunk 2]
```diff
@@ -4,14 +4,47 @@
...
-@Table(name="users")
+@Table(name="strategies")
+
+@NamedQueries(
+        {
+                @NamedQuery(name="strategies.getActive",
+                        query="select strategy from Strategy as strategy where strategy.isRunning = true",
+                        hints = {@QueryHint(name = "org.hibernate.cacheable", value = "true")}),
+                @NamedQuery(name="strategies.getTickers",
+                        query="select strategy.stock from Strategy as strategy",
```
Comment (KetanReddy) [inline apolloServer/src/main/java/com/apollo/com/apollo/entities/Strategy.java]: Will this return all unique or just all?

File: apolloServer/src/main/java/com/apollo/entities/Trade.java [hunk 3]
```diff
@@ -1,10 +1,18 @@
...
+package com.apollo.entities;
 import javax.persistence.*;
 import java.io.Serializable;
-import java.util.Date;
+import java.sql.Timestamp;
 
 @Entity
 @Table(name="trades")
+
+@NamedQueries(
```
Comment (KetanReddy) [inline apolloServer/src/main/java/com/apollo/entities/Trade.java]: delete

**Pull request merged**